systemd/coreos-installer-secure-ipl-reboot.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
+Requires=coreos-installer.target
+After=coreos-installer.target
+OnFailure=emergency.target
+OnFailureJobMode=replace-irreversibly
+ConditionPathExists=/run/coreos-installer-reboot
+ConditionKernelCommandLine=coreos.inst.secure_ipl
+
+[Service]
+Type=simple
+ExecStart=/bin/sh -c '[ -e /dev/vmcp ] && vmcp ipl loaddev || systemctl --no-block reboot'
```

Comment:
Minor: a comment here about z/VM and LPAR would be useful. E.g.

> On z/VM, we need to use `vmcp ipl loaddev`. Otherwise, for the LPAR case, we can just reboot as usual.

?

Reply:
Done